### PR TITLE
Fix docs for renderBufferImmediate

### PR DESCRIPTION
**Pull request by martinRenou**
Looking at the code I saw that it actually does not take a Material as input. I suppose it's a leftover from an old implementation? Also, I guess there was typo: `renderImmediateObject` -> `renderObjectImmediate`

See https://github.com/mrdoob/three.js/blob/9cd486bd4fc2f19d2817ed5f93ac0cf736dfe43c/src/renderers/WebGLRenderer.js#L657

### CODE REVIEW

**Comment by martinRenou** [comment]
I also see there is no TypeScript definition of `renderObjectImmediate`, and the only mention I found for this function was in there and it does not give any information about. Is it a kind-of private API?

**Comment by Mugen87** [comment]
Can you also apply the fix for the Chinese version?

https://threejs.org/docs/index.html#api/zh/renderers/WebGLRenderer.renderBufferImmediate

File: docs/api/en/renderers/WebGLRenderer.html [hunk 1]
```diff
@@ -427,12 +427,11 @@ <h3>[method:null render]( [param:Scene scene], [param:Camera camera] )</h3>
...
 		<p>Render a buffer geometry group using the camera and with the specified material.</p>
 
-		<h3>[method:null renderBufferImmediate]( [param:Object3D object], [param:shaderprogram program], [param:Material shading] )</h3>
+		<h3>[method:null renderBufferImmediate]( [param:Object3D object], [param:shaderprogram program] )</h3>
 		<p>object - an instance of [page:Object3D]<br />
 		program - an instance of shaderProgram<br />
-		shading - an instance of Material<br /><br />
 
-		Render an immediate buffer. Gets called by renderImmediateObject.
+		Render an immediate buffer. Gets called by renderObjectImmediate.
```
Comment (WestLangley) [inline docs/api/en/renderers/WebGLRenderer.html]: What is an immediate buffer?

Comment (Mugen87) [reply docs/api/en/renderers/WebGLRenderer.html]: Maybe like so:
```
Renders an instance of [page:ImmediateRenderObject]. Gets called by renderObjectImmediate().
```

Comment (martinRenou) [reply docs/api/en/renderers/WebGLRenderer.html]: Will do! I won't be able to apply this particular change in Chinese though.

**Comment by martinRenou** [comment]
`shaderProgram` also seems to not be documented. Is it public API?

**Comment by Mugen87** [comment]
You can refer to [WebGLProgram](https://threejs.org/docs/index.html#api/en/renderers/webgl/WebGLProgram).

**Review by Mugen87 — APPROVED** [review]

**Comment by sciecode** [comment]
I think we might need to update all `WebGL*` pages, as most of them look outdated. 

The current constructor info on `WebGLProgram` docs is no longer correct.

https://github.com/mrdoob/three.js/blob/b11f897812a8a48bcd81e9bd46785d07939ec59e/src/renderers/webgl/WebGLProgram.js#L378

https://github.com/mrdoob/three.js/blob/e3d60b17c0497a08ecb166eae799eb5cf6d9e7f5/docs/api/en/renderers/webgl/WebGLProgram.html#L108-L109

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**